### PR TITLE
reposync: Use a unique temp dir for cache

### PR DESCRIPTION
**Pull request by gbenhaim**
Repo injection to reposync-config.repo can create
cache issue which causing reposync to fail.

In order to avoid this, for each invocation of 'lago ovirt reposetup',
use a unique temp cache dir.

Signed-off-by: gbenhaim <galbh2@gmail.com>

### CODE REVIEW

**Comment by gbenhaim** [comment]
ci test please

**Comment by gbenhaim** [comment]
verified
without the patch http://jenkins.ovirt.org/view/oVirt%20system%20tests/job/ovirt-system-tests_manual-gal/1/

with the patch http://jenkins.ovirt.org/view/oVirt%20system%20tests/job/ovirt-system-tests_manual-gal/2/

File: ovirtlago/reposetup.py [hunk 1]
```diff
@@ -218,3 +222,6 @@ def sync_rpm_repository(repo_path, yum_config, repos):
                             'for repoid: %s, aborting.'
                         ) % repo
                     )
+
+        LOGGER.debug('Removing temp cache dir {}'.format(tmp_cache_dir))
+        shutil.rmtree(tmp_cache_dir)
```
Comment (nvgoldin) [inline ovirtlago/reposetup.py]: On exception - this will not be removed, is that intentional?

**Comment by gbenhaim** [comment]
ci merge please

**Pull request merged**